components/layout/blog-three-column/blog-three-column.hbs

```diff
@@ -0,0 +1,77 @@
+<div class="section--bg-white">
+    <div class="grid-col-content__three section__inner">
+
+        <a href="#" class="content-block--clickable content-block--blog">
```

Comment:
Could --blog just extend --clickable since we would never have one of these that shouldn't be clickable, right?